.github/workflows/client-tests.yml

```diff
@@ -5,7 +5,6 @@ jobs:
   test-client-ubuntu:
     name: "Client Tests"
     runs-on: ${{ matrix.os }}
-    if: github.event.pull_request.draft == false
```

Comment:
We don't want to run these tests on draft, so can you enable this?

Reply:
We decide to leave this in, because of the budget concerns but add a note in the contributing guide. We might consider a better solution. Like canceling previous running builds on non `develop` branch